pandas/tests/extension/arrow/arrays.py

```diff
@@ -67,6 +68,26 @@ def construct_array_type(cls) -> Type["ArrowStringArray"]:
         return ArrowStringArray
 
 
+@register_extension_dtype
```

Comment:
can you put these in the test file for now as I am not sure we agree on these names (and is just used for testing ATM).

Reply:
I can move them but I wanted to keep the dtype here as done for the other test-Arrow-dtypes.

Reply:
Done, CI passed except the Docs but the warning about missing sparse methods are unrelated to this PR.